runtime/gc_base/FinalizableClassLoaderBuffer.hpp

```diff
@@ -88,9 +88,9 @@ class GC_FinalizableClassLoaderBuffer
 	 */
 	GC_FinalizableClassLoaderBuffer(MM_GCExtensions *extensions) :
 		_head(NULL)
-		,_tail(NULL)
-		,_count(0)
-		,_extensions(extensions)
+		, _tail(NULL)
+		, _count(0)
+		, _extensions(extensions)
 	{}
```

Comment:
I think this pattern is an improvement: it reads better, additions or removals (at the end) don't change any other lines, but I suggest the first field should line up with the rest by moving the colon:
```c
	GC_FinalizableClassLoaderBuffer(MM_GCExtensions *extensions)
		: _head(NULL)
		, _tail(NULL)
		, _count(0)
		, _extensions(extensions)
	{}
```